redpanda/templates/configmap.yaml

```diff
@@ -67,10 +70,13 @@ data:
       superusers: {{ toJson $users }}
   {{- end }}
   {{- with (dig "cluster" dict .Values.config) }}
-    {{- toYaml . | nindent 6 }}
+      {{- toYaml . | nindent 6 }}
   {{- end }}
   {{- with (dig "tunable" dict .Values.config) }}
-    {{- toYaml . | nindent 6 }}
+      {{- toYaml . | nindent 6 }}
+  {{- end }}
+  {{- if not (hasKey "storage_min_free_bytes" .Values.config.cluster) }}
+      storage_min_free_bytes: {{- include "storage-min-free-bytes" . }}
```

Comment:
Bear in mind that when setting this in redpanda.yaml it'lll only take effect the first time you start up (this is the legacy upgrade path for importing pre-22.1 style configs)

Reply:
It's undocumented, so how would one maintain this after the initial configuration?

Reply:
Cluster vs. node configuration: https://docs.redpanda.com/docs/cluster-administration/configuration/

redpanda.yml is for node config (the file is authoritative).  For cluster configuration, redpanda's internal store is authoritative.  There is a one-time read of cluster configuration properties from redpanda.yaml to accomodate upgrades from versions <22.1 where redpanda.yaml was for everything and there was no concept of cluster-wide properties.

Helm and ansible need updating as/when we have people available to do it -- operator already speaks the cluster config API (see calls to `PatchClusterConfig` for a thread to pull on).

I'm not suggesting you implement support for cluster configuration in this PR, just flagging it as a caveat for anywhere that the redpanda.yaml upgrade path is used in helm/ansible.

Reply:
Added the rpk command to the post-upgrade job.